requirements_dev.txt

```diff
@@ -1,6 +1,8 @@
 -r requirements.txt
 coveralls>=1.3.0
-flake8==3.5.0
+flake8==3.7.9
```

Comment:
Minor: is it worth updating requirements_dev.txt in AWSM at the same time, which is still 3.5.0?

Reply:
I think that is what we are trying to figure out with dependancy testing. Since these are just dev requirements and not in the final docker image, I don't see this as a big problem yet. We can do greater than 3.5.0 if that's the case

Reply:
You could even think about removing the version string all together. Like you said, won't break anything

Reply:
Updated, there was a conflict with autopep8 but greater than should fix that